packages/jwt-verifier/test/util.js

```diff
@@ -49,25 +52,28 @@ function getAccessToken(options = {}) {
         state: 'foo'
       }
       const authorizeUrl = ISSUER + '/v1/authorize?' + qs.stringify(authorizeParams);
-      request.get(authorizeUrl, {followRedirect: false}, function(err, resp, body) {
-        const statusCode = resp ? resp.statusCode : null;
-        if (err || (resp && resp.statusCode >= 400) || !resp) {
-          console.error('/api/v1/authorize returned error: ', statusCode, err || body);
-          return reject(new Error('/api/v1/authorize error: ' + statusCode, + ' ' + err || body));
-        }
 
-        const parsedUrl = url.parse(resp.headers.location, true);
-        if (parsedUrl.query.error) {
-          console.error('/api/v1/authorize error in query:', parsedUrl.query.error);
-          return reject(parsedUrl.query.error);
-        }
-        const match = resp.headers.location.match(/access_token=([^&]+)/);
-        const accessToken = match && match[1];
-        if (!accessToken){
-          return reject(new Error('Could not parse access token from URI'));
-        }
-        resolve(accessToken);
-      })
+      return fetch(authorizeUrl, { redirect: 'manual' });
+    }).then(resp => {
+      if (!resp.status >= 400) {
```

Comment:
Should this be `(resp.status >= 400)` ? Without the `!` ?

Reply:
Good catch! Thanks